instana-java-sdk/src/main/java/com/instana/sdk/support/SpanSupport.java

```diff
@@ -96,22 +120,40 @@ public static long currentTraceId(Span.Type type, String name) {
   }
 
   /**
-   * @return The current trace ID for the currently recorded {@link Span}. If no span is currently recorded,
-   *         {@code null} is returned.
-   * @param type
-   *          span type.
-   * @param name
-   *          span name.
+   * Provides the trace ID of the currently recorded span, regardless of its {@code type} or {@code name}.
+   * This can be an SDK or AutoTrace span. If no span is currently recorded, or if the Instana agent is not active,
+   * {@code null} is returned.
+   *
+   * @return trace ID of the currently recorded span,
+   * or {@code null} if no span is recorded or the Instana agent is not active
+   * @since 1.2.0
+   */
+  public static String traceId() {
```

Comment:
In SDK 2.0, when we will drop Java 6 and 7, we should use Optionals a bunch :D

Reply:
Hehe, yes we should :-)